### PR TITLE
pyiron: Require pyiron_base 0.1.16

### PR DESCRIPTION
**Pull request by jan-janssen**

### CODE REVIEW

File: pyiron/pyiron.xml [hunk 1]
```diff
@@ -2,6 +2,7 @@
     <description>meta tools</description>
     <requirements>
         <requirement type="package" version="0.3.2">pyiron</requirement>
+        <requirement type="package" version="0.1.16">pyiron_base</requirement>
```
Comment (bgruening) [inline pyiron/pyiron.xml:5]: why is this needed, shouldn't pyiron not get the pyiron_base package?

**Comment by jan-janssen** [comment]
It does, but I just updated pyiron_base to 0.1.16 previously it was 0.1.15 so to enforce the installation of pyiron_base 0.1.16 I added it as explizit dependency.

**Comment by bgruening** [comment]
I have fixed this locally. Thanks.
